molo/surveys/models.py

```diff
@@ -164,8 +160,9 @@ class MoloSurveyPage(
     )
     your_words_competition = BooleanField(
         default=False,
-        verbose_name='Is YourWords Competition',
-        help_text='This will display the correct template for yourwords'
+        verbose_name='GCN Yourwords Competition',
```

Comment:
@Mitso this should be happening in Springster and not molo surveys as it will appear for all the projcts